README.md

```diff
@@ -1,31 +1,40 @@
-# Node essentials
+# Node Essentials
```

Comment:
Content teams uses sentence case which means only the first word and proper nouns are capitalized.